randovania/games/dread/assets/lua/bootstrap_locations.lua

```diff
@@ -0,0 +1 @@
+for n,i in pairsTEMPLATE("pairs")do RL.Pickups[i]=RandomizerPowerup.PropertyForLocation(TEMPLATE("location")..n) end
```

Comment:
should this not be `pairs(TEMPLATE("pairs"))`? or am I missing something

Reply:
In Lua `func({...})` is the same as `func{...}`, which I think is the case here?

Reply:
Ah, I could move on of those `{}` to the lua if you like.
Currently it's done via 
`replacements["pairs"] = "{{{}}}".format(",".join(entries))`
so it creates something like
`for n,i in pairs{Item_EnergyTank001=4, SHORTENED item_missiletankplus_000=16,item_energyfragment_000=19}do RL.Pickups[i]=RandomizerPowerup.PropertyForLocation('s010_cave_'..n) end`

Reply:
ah okay so it's done this way to save a couple bytes? :+1:

Reply:
moving the outermost pair of braces into the Lua file would be a bit more readable yeah

Reply:
Done